### DISCUSSION TITLE
Add more MPPTs

### DISCUSSION THREAD

**Discussion by schmue81** [slipx06/sunsynk-power-flow-card#291] [Ideas]
I have a pv with 7 MPPTs but only 4 are supported.
Can you please increase them?

**Comment by pnSys** [comment]
A few additional MPPTs could be very useful. to monitor two systems on one card.

**Comment by slipx06** [comment]
There is not enough space on the full card, but might be possible of the lite or compact version

**Comment by schmue81** [comment]
This could be a compromise. A wide card verison would be also nice as mentioned in another topic
